specs/UserActivityHistory/UserActivityHistoryAPI.md

```diff
@@ -0,0 +1,243 @@
+UserActivityHistory
+===
+
+# Background
+
+The UserActivity class can be used to note down and preserve a record of activities that the user
+is currently doing on their computer - e.g., browsing a website, reading a Word document, etc.
+
```

Comment:
"This allows Windows to have insight into the application state, enabling smart experiences that are built around the semantics of the app. For example, a document editor can give Windows information about the document that the user is editing, so that Recall can later take the user to the document at the same location."